libs/designsystem/src/lib/components/data-table/table/table.component.scss

```diff
@@ -0,0 +1,63 @@
+@use '~@kirbydesign/core/src/scss/utils';
```

Comment:
Related to this whole file: 

As it might be evident from the following comments, @jkaltoft and I have quite an opinionated take on this 😅 If it is just too much information overload, I have made a quick pastebin with the suggested changes so it is hopefully easier to grasp than my ramblings: https://pastebin.com/raw/5puhQCJs